posts/Ippon-USA-offers-a-helping-hand-to-those-impacted-by-natural-disasters.md

```diff
@@ -0,0 +1,32 @@
+ 
+Ippon USA offers a helping hand to those impacted by natural disasters. 
+ 
+“Let us remember: One book, one pen, one child, and one teacher, can change the world. Never doubt that a small group of thoughtful committed citizens can change the world. Indeed, it is the only thing that ever has.” - Malala Yousafzai
+ 
+About Me:
+My name is Danny Collins and I work for Ippon Technologies USA as a senior project manager. I have 14 years experience in the field of change and project management across multiple sectors on several continents. 
+Firstly, I am extremely fortunate to have a transferable skill set as a result of my career in project management, that can be used to build schools in rural Mexico as well as being able to manage a complex data migration in the abyss of the big data/cloud consulting world.
+Secondly, I am grateful to work for such a socially conscious and empathetic organisation as Ippon Technologies USA who have made my volunteering with All Hands & Hearts - Smart Response possible by allowing me the scope and flexibility to go off and help others, at the temporary expense of our deliverables here in the United States. Shout out to Romain Lheritier, our Managing Director and Jamie Bock, our Head of HR for not laughing me out of the room when I pitched this to them back in November. 
+Background:
+
+On September 7, 2017, one of the most lethal earthquakes in Mexican history hit close to the southern state of Chiapas, with a devastating magnitude of 8.2. Just 12 days later and a few hundred miles away, a 7.1 magnitude quake rocked central Mexico, toppling buildings, breaking gas mains, knocking out electricity and sparking fires across the city and other towns in central Mexico.
```

Comment:
Consider adding images like maps, or pictures from your trip.
Images are added like so:

![Earthquake Map](https://link-to-image)